scripts/resolve_suggestions_urls.py

```diff
@@ -26,18 +26,18 @@ def json_path_to_pointer(json_path):
 
 
 def find_suggestion_urls(data):
-    json_path = parse("$..suggestions_url")
+    json_path = parse("$..suggestions")
     for match in json_path.find(data):
         suggestions_url = match.value
-        if "suggestions_url_root" in suggestions_url:
+        if "suggestions_url_root" in suggestions_url["url"]:
```

Comment:
`suggestion_url` at the moment is the object.
```py
        suggestions_url = match.value["url"]
        if "suggestions_url_root" in suggestions_url:
```

Reply:
Sorry, fixed now, thanks!

Reply:
Pushed now!